src/main/java/com/epam/parso/impl/SasFileParser.java

```diff
@@ -1206,8 +1204,8 @@ public void processSubheader(long subheaderOffset, long subheaderLength) throws
                 int textSubheaderIndex = bytesToShort(vars.get(0));
                 int columnNameOffset = bytesToShort(vars.get(1));
                 int columnNameLength = bytesToShort(vars.get(2));
-                columnsNamesList.add(columnsNamesStrings.get(textSubheaderIndex).substring(columnNameOffset,
-                        columnNameOffset + columnNameLength).intern());
+                columnsNamesList.add(bytesToString(Arrays.copyOfRange(columnsNamesBytes.get(textSubheaderIndex),
```

Comment:
Do we need Arrays.copyOfRange here? Don't you think "new String(columnsNamesBytes.get(textSubheaderIndex), columnNameOffset, columnNameLength, encoding).intern()" would be better here?